external/CMakeLists.txt

```diff
@@ -59,4 +58,40 @@ if (${ENABLE_SPIRV_CODEGEN})
       message(FATAL_ERROR "GoogleTest was not found - required for SPIR-V codegen")
     endif()
   endif()
-endif()
\ No newline at end of file
+
+  # We need Effcee for SPIR-V codegen testing.
+  if (${SPIRV_BUILD_TESTS})
+    set(DXC_RE2_DIR "${DXC_EXTERNAL_ROOT_DIR}/re2" CACHE STRING "Location of RE2 source")
+    set(DXC_EFFCEE_DIR "${DXC_EXTERNAL_ROOT_DIR}/effcee" CACHE STRING "Location of Effcee source")
+
+    if (IS_DIRECTORY ${DXC_RE2_DIR})
+      # Silence some level-3 and level-4 warnings from RE2.
+      add_compile_options(/wd4201 /wd4815 /EHs)
```

Comment:
Now we don't need `/wd4201`, `/wd4815` anymore with https://github.com/google/re2/commit/e7efc48dbbb878764af3df3a997535469186aecd.